### PR TITLE
FOSS4G2022 slides: initial updates

### PR DESCRIPTION
**Pull request by neteler**
- selected cleanup (more to come)
- added new slides:
    - Student grant program
    - Automated releases from GitHub
- some screenshots added

Sources:
- https://github.com/wenzeslaus/grass-gis-talks/blob/main/doi2022.html
- https://github.com/veroandreo/grass-gis-talks/blob/main/wageningen2021.html
- own material

### CODE REVIEW

**Review by veroandreo — APPROVED** [review]
:+1:

File: foss4g2022.html [hunk 1]
```diff
@@ -250,6 +276,201 @@ <h2>All-in-one software suite</h2>
...
+</section>
+
+
+<section>
+    <h2>Integration with Jupyter Notebooks</h2>
+    <p>
+        Time-series visualization
+    </p>
+    <img src="img/jupyter_timeseries_viewshed.gif" class="stretch">
+    <p>by Vaclav Petras and Anna Petrasova</p>
```
Comment (veroandreo) [inline foss4g2022.html]: Add Caitlin as well, as this was the topic of her student grant

File: foss4g2022.html [hunk 2]
```diff
@@ -250,6 +276,201 @@ <h2>All-in-one software suite</h2>
...
+                <li>image classification can use sematic labels to identify bands</li>
+                <li>different scenes can then be classified if semantic labels are the same</li>
+            </ul>
+        </li>
+        <li>
+            organize bands within one spatio-temporal dataset
+        </li>
+    </ul>
+    <br>
+    <img src="img/i_superpixels_slic_ndvi.png" class="stretch">
```
Comment (veroandreo) [inline foss4g2022.html]: replace this image with eg https://github.com/neteler/grass-gis-talks-markus/blob/main/img/band_references_scheme.png

File: foss4g2022.html [hunk 3]
```diff
@@ -250,6 +276,201 @@ <h2>All-in-one software suite</h2>
...
+            Veronica&nbsp;Andreo (CONICET),
+            Stefan&nbsp;Blumentrath (Norwegian Institute for Nature Research - NINA)
+            -->
+            &hellip;
+        </small>
+    </p>
+</section>
+
+<section>
+    <h2>Integration of results from Google Summer of Code 2021</h2>
```
Comment (veroandreo) [inline foss4g2022.html]: Jupyter integration was also part of the GSoC 2021

File: foss4g2022.html (hunk 1 above)
Comment (neteler) [reply foss4g2022.html]: Sure, done in a61338f20952977bc9f9ca41a78e78390a419b73

File: foss4g2022.html (hunk 2 above)
Comment (neteler) [reply foss4g2022.html]: yes, 64dce78f2e15f32015b9c3f081d3583e4a1d6c86

File: foss4g2022.html (hunk 3 above)
Comment (neteler) [reply foss4g2022.html]: Yes, added in 73a18ba83d91e6c0f7412d792824add4d61587e1

**Comment by neteler** [comment]
After a MAJOR reshuffling of slides I'll merge this now as an interim version. More to come in a new PR.

**Pull request merged**